kubetest/e2e.go

```diff
@@ -393,6 +436,8 @@ type deployer interface {
 	IsUp() error
 	SetupKubecfg() error
 	Down() error
+	FederationUp() error
```

Comment:
Does deploy the federation control plane change depending on how the underlying clusters are deployed?

If not then it suggests we can just add FederationUp / FederationDown phases that always call federation/cluster/federation-up.sh.

Or is the idea that kubernetes-anywhere and kops will eventually have their own federation deployment strategies?